source/sass/patterns/organisms/site-header.scss

```diff
@@ -23,72 +20,65 @@ $siteHeader-block_end_space: 7px !default;
   @include max-inline-size($grid-max_width);
 
   position: relative;
-  @supports (display:flex) {
+  @supports (display: flex) {
     position: static;
     display: flex;
     align-items: center;
   }
 
   --block-size: #{$siteHeader-block_size__default};
+  --inline-space: #{$siteHeader-inline-space__default};
   @include block-size($siteHeader-block_size__default);
+  @include padding($siteHeader-inline-space__default, inline);
 
   @include mq($from: medium) {
     --block-size: #{$siteHeader-block_size__medium};
+    --inline-space: #{$siteHeader-inline-space__medium};
     @include block-size($siteHeader-block_size__medium);
+    @include padding($siteHeader-inline-space__medium, inline);
+  }
+
+  @include mq($from: x-wide) {
+    --inline-space: 0;
+    @include padding(0, inline);
   }
 
   @supports (--name: value) {
     @include block-size(var(--block-size));
+    @include padding(var(--inline-space), inline);
   }
 }
 
 .site-header__logo_link {
 
+  position: relative;
+  top: 50%; // stylelint-disable-line csstools/use-logical
+  transform: perspective(1px) translateY(-50%);
+  @supports (display: flex) {
+    position: static;
+    transform: none;
```

Comment:
I think it needs `flex-grow: 1;` here to ensure SVGs appear when they don't have specified internal `width` and `height` attributes.

Reply:
Tried various combinations, but the link ends up filling the header due to the `display: block` on the image. There'll be a way to do it, but suggest we follow it up.